api/commands/build.ts

```diff
@@ -94,7 +94,7 @@ export class Builder {
     return {
       ...this.toPartialExecutionAction(table),
       type: "table",
-      tableType: table.type,
+      tableType: utils.tableEnumTypeToString(utils.tableTypeFromProto(table, true)),
```

Comment:
To match `type` above, can't this just be `dataform.TableType.TABLE`

Reply:
No, `type` corresponds to `TableMetadata.Type` https://github.com/dataform-co/dataform/blob/f61bcedfd4d84f667a711c414fa868eafa8bcc07/protos/execution.proto#L181, it's slightly different from `typleType` options from `TableType` here https://github.com/dataform-co/dataform/blob/b25786b3a950921dbd58bc71236a0f83b8a0879f/protos/core.proto#L162